app/src/main/java/com/arctouch/io/outdoorsychallenge/domain/repository/VehicleRepository.kt

```diff
@@ -1,11 +1,18 @@
 package com.arctouch.io.outdoorsychallenge.domain.repository
 
+import com.arctouch.io.outdoorsychallenge.data.source.database.IVehicleDatabaseSource
 import com.arctouch.io.outdoorsychallenge.data.source.remote.outdoorsy.IVehicleRemoteDataSource
+import com.arctouch.io.outdoorsychallenge.domain.model.Vehicle
+
+class VehicleRepository(private val dataSource: IVehicleRemoteDataSource,
+                        private val databaseSource: IVehicleDatabaseSource) : IVehicleRepository {
```

Comment:
the correct chopdown would be breaking a pine before the firs parameter and after the last parameter